code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -56,7 +56,7 @@
 		if(throwing) // you keep some momentum when getting out of a thrown closet
 			step(AM, dir)
 	if(throwing)
-		throwing = 0
+		throwing.finalize(FALSE)

```

Comment:
Can `dump_contents` even ever be called and the things inside it are being thrown?

Hmm, now that I think of it, probably not for us, but with how TG's explosion system works vastly different to ours...